main.css

```diff
@@ -108,3 +121,2555 @@ body {
       font-size: 0.875rem; /* 14px */
       font-weight: 600;
     }
+
+.prog-bar{}
+  .prog-bar__button{
+    margin: 0.75rem 1.5rem;
+  }
+  .prog-bar__button-dropdown{
+    -webkit-appearance: none;
+    -moz-appearance: none;
+    appearance: none;
+    padding: 0.5rem 2.5rem;
```

Comment:
I think it looks better with:
`padding: 0.5rem 2.5rem 0.5rem 0.5rem;`
Making this change has the following effect from how it is now:
![screen shot 2017-03-13 at 15 05 44](https://cloud.githubusercontent.com/assets/7805691/23860353/9a77deb6-07fe-11e7-8922-26fb8099630a.png)
to this:
![screen shot 2017-03-13 at 15 05 31](https://cloud.githubusercontent.com/assets/7805691/23860359/a0609958-07fe-11e7-9da4-3e2c75dbb5bb.png)